volume/ceph/ceph.go

```diff
@@ -177,22 +186,56 @@ func (v *Volume) Mount(id string) (mappedDevicePath string, returnedError error)
 	if err != nil {
 		return "", err
 	}
+
+	if fsType == criptoLUKS {
+		cmd = exec.Command("cryptsetup", "luksOpen", v.mappedDevicePath, v.Name())
```

Comment:
Let's be nice to cryptsetup, and pass "-q" so it knows we're in batch mode. You might also want to pass "--key-file=-" to explicitly get the key from stdin in a programmatic way.